ocp-templates/mpp/backend.yaml

```diff
@@ -174,6 +174,24 @@ objects:
     to:
       kind: Service
       name: ibutsu-backend
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    annotations:
+      description: A route to the backend
+    labels:
+      app: ${APP_NAME}
+      shard: internal
+    name: ibutsu-backend
+    namespace: ${NAMESPACE}
+  spec:
+    host: ${BACKEND_ROUTE_CNAME}
+    tls:
+      insecureEdgeTerminationPolicy: Redirect
+      termination: edge
+    to:
+      kind: Service
+      name: ibutsu-backend
```

Comment:
why are we creating 2 routes ? shouldn't we just have the first one ? (the one that's used by the CNAME ?)